packages/java/endpoint/src/main/java/dev/hilla/crud/CrudService.java

```diff
@@ -0,0 +1,24 @@
+package dev.hilla.crud;
+
+import java.util.List;
+
+import org.springframework.data.domain.Pageable;
+
+/**
+ * A browser service that can create, read and update a given type of object.
```

Comment:
Something called "browser service" is communicating something else IMO. I suggest to change it to: 
```suggestion
 * A browser-callable service that can create, read and update a given type of object.
```